app/views/layouts/application.html.erb

```diff
@@ -78,6 +78,7 @@
     </div>
 
     <%= render partial: 'layouts/footer' %>
+    <%= render partial: 'layouts/javascript_variables' %>
```

Comment:
If these variables are only used on one page, should this be at the bottom of that page.